support-frontend/assets/components/ticker/contributionTicker.scss

```diff
@@ -1,4 +1,4 @@
-// -----  gu-sass ----- //
+@import '~stylesheets/gu-sass/gu-sass';
```

Comment:
Maybe add a note to say this is bc the component is in Storybook

Reply:
Nope! this import is also picked up by `support-frontend` [as normal](https://github.com/guardian/support-frontend/wiki/CSS) we started this migration a bit ago